src/test/java/com/rbkmoney/proxy/mocketbank/handler/IntegrationTest.java

```diff
@@ -36,10 +37,10 @@
 @Slf4j
 public abstract class IntegrationTest {
 
-    protected String invoiceId = "TEST_INVOICE" + (int) (Math.random() * 50 + 1);
-    protected String paymentId = "TEST_PAYMENT" + (int) (Math.random() * 50 + 1);
-    protected String recurrentId = "TEST_RECURRENT" + (int) (Math.random() * 500 + 1);
-    protected String refundId = "TEST_REFUND" + (int) (Math.random() * 500 + 1);
+    protected String invoiceId = "taLtB5DcAq";
+    protected String paymentId = "1";
+    protected String recurrentId = "1HyXdrtbm5Y";
+    protected String refundId = "1HyXfnApAoK";
```

Comment:
но зачем, если можно было оставить человекочитаемые? Я писал только про рандом

Reply:
Чтобы было понятно, как они выглядят в боевой среде

Reply:
Солидарен, что лучше оставить человекочитаемые варианты, думаю любой кто читает логи здесь, и так понимает как выглядят идентификаторы сущностей.
Предложил бы откатить до
```suggestion
    protected String invoiceId = "TEST_INVOICE";
    protected String paymentId = "TEST_PAYMENT";
    protected String recurrentId = "TEST_RECURRENT";
    protected String refundId = "TEST_REFUND";
```